apps/svelte.dev/content/blog/2025-04-01-whats-new-in-svelte-april-2025.md

```diff
@@ -0,0 +1,106 @@
+- The Svelte CLI also got an upgrade to its code generation for more intuitive formatting (**0.7.0**, [#380](https://github.com/sveltejs/cli/pull/380))
+- `eslint-plugin-svelte` v3 has been released. The main update is improved support for Svelte 5. Check out the [CHANGELOG](https://github.com/sveltejs/eslint-plugin-svelte/releases) for more details.
+
+There were also quite a few [fixes](https://github.com/sveltejs/language-tools/releases/tag/extensions-109.5.3) to the language tools earlier in the month - so be sure to keep your plugins up to date!
+
+
+## What's new in SvelteKit
+- The new action failure helper checks if a variable is an instance of `ActionFailure` (**2.8.0**, [Docs](https://svelte.dev/docs/kit/@sveltejs-kit#ActionFailure), [#12878](https://github.com/sveltejs/kit/pull/12878))
+- Vite 6 is now supported - making SvelteKit faster and more configurable at build time (**2.9.0**, [Blog Post](https://vite.dev/blog/announcing-vite6) and [#12270](https://github.com/sveltejs/kit/pull/12270))
+- The new server and client `init` hook makes it easier to inject context upon app initialization - helpful for i18n and more (**2.10.0**, Docs, [#13103](https://github.com/sveltejs/kit/pull/13103))
+- The new `$app/state` module replicates the existing `$app/stores` module but using Svelte 5 state (**2.12.0**, [Docs](https://svelte.dev/docs/kit/app-state), [#13140](https://github.com/sveltejs/kit/pull/13140))
+- A new configuration option, `bundleStrategy: 'split' | 'single' | 'inline'`, lets you configure how your app’s JavaScript and CSS files are loaded (**2.13.0/2.15.0**, [Docs](https://svelte.dev/docs/kit/configuration#output), [#13173](https://github.com/sveltejs/kit/pull/13173) and [#13193](https://github.com/sveltejs/kit/pull/13193))
+- Hash-based routing is now built in and can be configured app-wide (**2.14.0**, [Docs](https://svelte.dev/docs/kit/configuration#router), [#13191](https://github.com/sveltejs/kit/pull/13191))
+- A custom identifier can now be invalidated within a `goto()` (**2.16.0**, [Docs](https://svelte.dev/docs/kit/app-navigation#goto), [#13256](https://github.com/sveltejs/kit/pull/13256))
+- The `postinstall` script has been removed to support pnpm 10 - users should add `"prepare": "svelte-kit sync"` to their package.json in order to avoid a warning from Vite (**2.16.0**, [#13304](https://github.com/sveltejs/kit/pull/13304))
+- Values for `cache-control` and `content-type` headers will now be validated in dev mode (**2.17.0**, [#13114](https://github.com/sveltejs/kit/pull/13114))
+- Server-side route resolution is now supported - improving performance and security across the board. See PR for more details (**2.17.0**, [Docs](https://svelte.dev/docs/kit/configuration#router), [#13379](https://github.com/sveltejs/kit/pull/13379))
```

Comment:
Are you doing more than just a month? Some of this stuff is from last year
```suggestion
```

Reply:
ahh! my bad! I think my scroll position might have been off when i was going through the changelog thanks for catching that!